### PR TITLE
documentation: ZENKO-1721_Lifecycle_Metrics

### PR DESCRIPTION
**Pull request by wabernat**
This PR adds the "Querying Lifecycle Events" section. This is the closest thing to metrics that we currently offer.

This fixes #ZENKO-1721 (until the Metrics feature is developed)

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e approve

**Review by tmacro — APPROVED** [review]

File: docs/docsource/Operation-Architecture/Zenko_CLI/index.rst [hunk 1]
```diff
@@ -10,15 +10,18 @@ Enabling command-line interactions enables programmatic access to
...
 
    CRR Metrics and Healthcheck <CRR_Metrics_and_Health>
    CRR Retry <CRR_Retry>
    CRR Pause and Resume <CRR_Pause_&_Resume>
    CRR Statistics <CRR_statistics>
    Object Lifecycle Management <Object_Lifecycle_Management>
 
-Accessing Zenko from the command line requires the following setup tasks.
+Accessing Zenko from the command line requires setting up access to 
+the S3 API, the Backbeat API, or both.
```
Comment (bennettbuchanan) [inline docs/docsource/Operation-Architecture/Zenko_CLI/index.rst:22]: Because this is unrelated to the scope of this PR, it'd be helpful to at least make it a separate commit, if not a separate PR. That can help for maintainability, e.g. when looking at the history of commits we can understand the intention of changes better, or in case a commit needs to be reverted, etc.

It's a minor point though, so definitely no need to hold back this PR for it.

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst [hunk 2]
```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
...
+Zenko controller and S3 API functionality configured as described in
+:ref:`S3 API config`. Once this is configured, use the head-object command
+as described in
+https://docs.aws.amazon.com/cli/latest/reference/s3api/head-object.html.
+
+For example:
+
+.. code::
+   
+   $ aws s3api head-object --bucket mybucket-1 --key myobjects1 --endpoint http://10.0.0.1
```
Comment (bennettbuchanan) [inline docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: Let's make the example even more generic:

~mybucket-1~ `<bucket-name>`

~myobjects1~ `<key-name>`

~http://10.0.0.1~ `<endpoint-url>`

Note: I'm not aware of any standard pattern around examples in the docs, so please ignore this if it doesn't correspond to that.

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst [hunk 3]
```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
...
+
+.. code::
+
+   {
+      "AcceptRanges": "bytes",
+      "ContentType": "application/octet-stream",
+      "LastModified": "Tue, 16 Apr 2019 22:12:33 GMT",
+      "ContentLength": 1,
+      "ETag": "\"e358efa489f58062f10dd7316b65649e\"",
+      "StorageClass": "aws-backend",
```
Comment (bennettbuchanan) [inline docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: ~aws-backend~ `aws-storage-location`

That way we're more consistent with the naming scheme in Orbit which terms this a "storage location".

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst [hunk 4]
```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
...
+      "Metadata": {}
+   }
+
+The returned information describes the myobjects1 object in the mybucket-1 
+bucket. The StorageClass information indicates the object has transitioned to
+the aws-backend storage class (as a cloud service is defined in Zenko's
```
Comment (bennettbuchanan) [inline docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: ~to the aws-backend storage class (as a cloud service is defined in Zenko's interpretation of the S3 protocol)~ to a storage location named "aws-storage-location", as defined by the Zenko user.

...or something along those lines 😄

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst [hunk 5]
```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
...
\ No newline at end of file
+objects in the specified bucket.
+
+.. _Lifecycle Queries:
+
+Querying Lifecycle Events
+-------------------------
+
+A lifecycle metrics feature is not available for release 1.1. The only
+available lifecycle event metric is to send a head-object request to
```
Comment (bennettbuchanan) [inline docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: I'm not sure I'd go so far as to characterize the head object response as a "metric". Maybe we can say something like "The storage location of the transitioned object data can be accessed by viewing the object's metadata, for example by making a head object request".

File: docs/docsource/Operation-Architecture/Zenko_CLI/index.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Zenko_CLI/index.rst:22]: stet

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst (hunk 2 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: done

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst (hunk 3 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: done

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst (hunk 4 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: done

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst (hunk 5 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: done

**Comment by wabernat** [comment]
@bert-e reset

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst [hunk 6]
```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
...
-objects in the specified bucket.
\ No newline at end of file
+objects in the specified bucket.
+
+.. _Lifecycle Queries:
+
+Querying Lifecycle Events
+-------------------------
+
+A lifecycle metrics feature is not available for release 1.1. You can,
```
Comment (bennettbuchanan) [inline docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: > A lifecycle metrics feature is not available for release 1.1. 

I think we can leave this sentence out because there is a chance this might change. If we are able to implement metrics, I think the doc here will apply nicely still. 😸

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst [hunk 7]
```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
...
+however, access the storage location of transitioned object data by
+viewing the object's metadata, for example by making a head-object
```
Comment (bennettbuchanan) [inline docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: ~head-object request~ HEAD request

"head-object" is a reference to the command provided by the user via the command line, while the request itself would be referred by the HTTP request method (i.e. HEAD).

For example see: https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectHEAD.html

**Review by bennettbuchanan — COMMENTED** [review]
Great! Just a couple more follow-up comments, then it LGTM!

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst (hunk 6 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: done

File: docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst (hunk 7 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst]: done.

**Review by bennettbuchanan — APPROVED** [review]
:shipit: Ship it!

**Comment by ssalaues** [comment]
@bert-e bypass_build_status
